README.md

```diff
@@ -87,22 +87,22 @@ sudo rm /usr/local/bin/eosio-*
 * eosio-objdump
 * eosio-readelf
 
-Below tools are not installed after brew install, you get them only by building the repository and installing from scracth, [see here](#guided_installation_or_building_from_scratch)
-eosio-abidiff
-eosio-ranlib
-eosio-ar
-eosio-objdump
-eosio-readelf
+Below tools are not installed after brew install, you get them only by building the repository and installing from scracth, [see here](#guided-installation-or-building-from-scratch)
```

Comment:
Sorry one more spelling issue that I just caught `scracth` instead of `scratch` . Worth checking the rest of the doc as well.

Reply:
Fixed.